.circleci/config.yml

```diff
@@ -298,15 +298,15 @@ jobs:
           command: git submodule update --init --recursive
       - restore_cache:
           keys:
-          - e2e-test-cache-{{ checksum "~/workbench/e2e/yarn.lock" }}
-          - e2e-test-cache-
+          - v1-e2e-test-cache-{{ checksum "~/workbench/e2e/yarn.lock" }}
```

Comment:
Adding a version number. From [CircleCI official doc](https://circleci.com/docs/2.0/caching/):
`If you would like to invalidate your cache manually, you can bump v1 to v2 in your config.yml file. In this case, you would now have a new cache key v2-npm-deps, which will trigger the storing of a new cache.`